.github/workflows/ci.yml

```diff
@@ -59,6 +59,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Install Chef
         uses: actionshub/chef-install@main
+        with:
+          version: 15.3.14
```

Comment:
Let's please not lock this into a specific version. Is there a reason why you did this?

Reply:
For Chef 17 it seem to fail somewhere inside the poise cookbook:
https://github.com/sous-chefs/consul/runs/2719646306

In this PR we raise the cookbook minimum version from 13 to 15.3
I thought it makes sense to pin testing to that particular version to test it.

Reply:
I think this is definitely something we should do long term, but also with Chef, 16 and 17 test suites. 

That said if this update doesn't work with 17 then it doesn't work.